coral/src/app/features/topics/overview/messages/TopicMessages.tsx

```diff
@@ -17,38 +23,69 @@ function isNoContentResult(
 
 function TopicMessages() {
   const { topicName } = useTopicDetails();
+  const [offset, setOffset] = useOffsetFilter();
   const {
     data: consumeResult,
     isLoading,
     isError,
+    isRefetching,
     error,
+    refetch: updateResults,
   } = useQuery({
-    queryKey: ["topicMessages"],
+    queryKey: ["topicMessages", offset],
     queryFn: () =>
       getTopicMessages({
         topicName,
         consumerGroupId: "notdefined",
         envId: "2",
-        offsetId: "5",
+        offsetId: offset,
       }),
-    keepPreviousData: true,
+    keepPreviousData: false,
```

Comment:
Nitpick: I think `keepPreviousData` is false by default so we may wan to remove this.

But also, maybe we want it to be `true` and add some info about the last time it was fetched? Like "Last fetched at XXXX".  It seems a little heavy to do that seemingly expensive call every time a user goes to this tab?
 

https://github.com/aiven/klaw/assets/20607294/7cc01c01-07a7-4cbf-9f11-eb88568942b2